static/js/calculate_score.js

```diff
@@ -1,16 +1,19 @@
 var resultJSON = "";
 var unusedTraitStudyArray = [];
-var traitObjects = []
-var studyObjects = [] //holds the study object so that their additional data (ethnicity, cohort, ect) can be accessed
 var traitsList = []
-var selectedStudies = []
 //if false, the VCF button is selected- used as a toggle to prevent action on double click
 var textButtonSelected = true;
 
 // ensure that they know they need to reupload the file by clearing the input area
 function pageReset() {
     var textInput = document.getElementById('input');
     textInput.value = ""
+
+    document.getElementById("database").checked = true;
+    document.getElementById("gwasDatabase").style.display = "initial";
+    document.getElementById("gwasUpload").style.display = "none";
+    
+    //TODO reset the GWAS upload input
```

Comment:
Don't forget this todo

Reply:
Ended up just removing it, I don't actually need to reset it